kibbeh/src/ui/SettingsIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from "react";
+
+export interface SettingsIconProps {
+  icon: ReactElement;
+  label: string;
+  trailingIcon?: ReactElement;
+  classes?: string;
```

Comment:
Default `classes` to empty string to avoid having `undefined` if no value is passed.